IBF-Typhoon-model/src/typhoonmodel/utility_fun/initialize.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+import cartopy
+from cartopy.io.shapereader import NEShpDownloader
+
+
+CARTOPY_SOURCE_TEMPLATE = 'https://naturalearth.s3.amazonaws.com/{resolution}_{category}/ne_{resolution}_{name}.zip'
+
```

Comment:
this file is 4mb we can also store it in the repo or store it on OneDrive maybe?
if this new link is a stable i am okay to keep it like this as well

Reply:
When I look at the downloaded files, I get that they are 16 MB. We could store in OneDrive, but S3 should be just as stable. And in fact, Cartopy just merged a PR to use the same URL! https://github.com/SciTools/cartopy/pull/1833
(we should keep the patch until the next release though).